docs/news/news-v5.0.x.rst

```diff
@@ -0,0 +1,282 @@
+Open MPI v5.0.x series
+======================
+
+This file contains all the NEWS updates for the Open MPI v5.0.x
+series, in reverse chronological order.
+
+Open MPI version 5.0.0rc3
+-------------------------
+:Date: 08 March 2022
+
+.. admonition:: MPIR API has been removed
+   :class: warning
+
+   As was announced in summer 2017, Open MPI has removed support of
+   MPIR-based tools beginning with the release of Open MPI v5.0.0.
+
+   The new PRRTE based runtime environment supports PMIx-tools API
+   instead of the legacy MPIR API for debugging parallel jobs.
+
+   see https://github.com/openpmix/mpir-to-pmix-guide for more
+   information
+
+
+.. admonition:: zlib is suggested for better user experience
+   :class: note
+
+   PMIx will optionally use zlib to compress large data streams.
+   This may result in shorter-than-normal startup times and
+   smaller memory footprints.  It is recommended to install zlib
+   and zlib-devel for a better user experience.
+
+- ORTE, the underlying OMPI launcher has been removed, and replaced
+  with PRTE.
+- New MPI-4.0 features:
+
+  - Added ``ERRORS_ABORT`` infrastructure.
+  - Added support for 'initial error handler'.
+  - Added error handling for 'unbound' errors to ``MPI_COMM_SELF``.
+  - Persistent collectives have been moved to the MPI namespace
+    from MPIX.
+  - Made ``MPI_Comm_get_info()``, ``MPI_File_get_info()``, and
+    ``MPI_Win_get_info()`` compliant to the standard.
+  - Added initial partitioned communication using persistent sends
+    and persistent receives.
+  - Added support for ``MPI_Info_get_string()``.
+  - Droped unknown/ignored info keys on communicators, files,
+    and windows.
+  - Added ``MPI_Isendrecv()`` and its variants.
+  - Added Support for ``MPI_Comm_idup_with_info()``.
+- Reworked how Open MPI integrates with 3rd party packages.
+  The decision was made to stop building 3rd-party packages
+  such as Libevent, HWLOC, PMIx, and PRRTE as MCA components
+  and instead 1) start relying on external libraries whenever
+  possible and 2) Open MPI builds the 3rd party libraries (if needed)
+  as independent libraries, rather than linked into libopen-pal.
```

Comment:
```suggestion

  .. note:: Open MPI no longer builds 3rd-party packages
      such as Libevent, HWLOC, PMIx, and PRRTE as MCA components
      and instead:
      
        #. Relies on external libraries whenever possible, and
        #. Builds the 3rd party libraries only if needed, and as independent libraries, rather than linked into the Open MPI core libraries.
```

Re-wrap and indent as appropriate -- it's hard to tell the correct indenting level here in a review comment.